test/test.py

```diff
@@ -0,0 +1,61 @@
+import boto3
+import botocore
+import json
+import os
+from emailgen import Emailgen
+import collections
+
+s3 = boto3.resource('s3', region_name = 'us-east-1')
+
+bucket_name = 'inbox-bucket-test'
+bucket_name1 = 'inbox-bucket-testresized'
+
+bucket = s3.Bucket(bucket_name)
+print("created source bucket\n")
+
+bucket1 = s3.Bucket(bucket_name1)
+print("created destination bucket\n")
+
+
+def upload(filename):
+  fileToBeUploaded = os.getcwd()+'/eml/'+filename
```

Comment:
We should not construct this one as it is already there at https://github.com/RealImage/extraction/pull/1/files#diff-485a45816c4cd61ca20e34c84f5cf564R40.

Reply:
The other one is for attaching the file to the mail, this one is basically to upload the created blob to the bucket